.gitignore

```diff
@@ -3,4 +3,5 @@ SavedState/
 **/*.flow
 **/*.log
 **/*.txt
-log*/
\ No newline at end of file
+log*/
+e2e/*.fail
```

Comment:
Seems that this is a leftover, since I don't see we are generating .fail files in the e2e folder.

Reply:
When you property based test fails it generates *.fail file that you can use to reproduce the case.